api/client/src/main/java/org/projectnessie/client/NessieConfigConstants.java

```diff
@@ -44,6 +44,44 @@ public final class NessieConfigConstants {
    * authentication.
    */
   public static final String CONF_NESSIE_AUTH_TOKEN = "nessie.authentication.token";
+
+  /**
+   * Required. The URL of the OAuth2 token endpoint, including the path to the token endpoint when
```

Comment:
Can you add the `{@value #CONF_...}` javadoc content? Helps to have the config key handy in docs (if we ever have some automatic way to generate a page out of this one).